tools/PI/DevHome.PI/Helpers/WatsonHelper.cs

```diff
@@ -130,12 +130,13 @@ public List<WatsonReport> GetWatsonReports()
     {
         Dictionary<string, WatsonReport> reports = [];
         EventLog eventLog = new("Application");
+        var targetProcessName = targetProcess.ProcessName;

```

Comment:
>; [](http://example.com/codeflow?start=57&length=1)

This one has me confused. We'd lose access to the process name if the targetprocess exited?

Reply:
No, this is the scenario, when the user goes to the Watson page and then immediately closes the bar window. WatsonHelper is still parsing the old watson reports at this moment and targetprocess object is disposed and set to null since we call TargetAppData.Instance.ClearAppData() on Close.